event_pubsub/event_repository.py

```diff
@@ -120,11 +120,12 @@ def insert_registry_event(self, block_number, event_name, json_str, processed, t
         self.connection.begin_transaction()
         try:
 
-            insert_query = "INSERT INTO registry_events_raw (block_no, event, json_str, processed, transactionHash, logIndex ,error_code,error_msg,row_updated,row_created) " \
+            insert_query = "INSERT INTO registry_events_raw (block_no, event, json_str, processed, transactionHash, logIndex, error_code, error_msg, row_updated, row_created) " \
                            "VALUES ( %s, %s, %s, %s, %s , %s, %s, %s, %s, %s ) " \
-                           "ON DUPLICATE KEY UPDATE row_updated=%s "
+                           "ON DUPLICATE KEY UPDATE logIndex=%s,  error_code=%s, error_msg=%s, row_updated=%s "
```

Comment:
Prashant, what happens to the new block number. Is that not persisted? I would like to track the new block number as well.

Reply:
We will keep track of the actual block no., will not keep track of uncle block.

Reply:
@prashantramangupta lets track the uncle block number as well. This is a schema change but will help us track and audit